demos/gallery/samples/common/CanvasWidget.js

```diff
@@ -0,0 +1,38 @@
+import { CanvasWidget } from "@hpcc-js/common";
+
+const data = [
+    ["A", 100],
+    ["B", 162],
+    ["C", 262]
+];
+class CanvasExample extends CanvasWidget{
+    constructor(){
+        super();
+    }
+    enter(domNode, element){
+        super.enter(domNode, element);
+        this.resize();
```

Comment:
Why is this needed?  Should it be in the CanvasWIdget?

Reply:
Hmm... evidently it isn't needed after all